packages/types/src/clerk.ts

```diff
@@ -673,6 +673,13 @@ export type SignUpRedirectOptions = RedirectOptions &
     initialValues?: SignUpInitialValues;
   };
 
+/**
+ * OrganizationIdentifier uniquely identifies an organization within an instance
+ */
+export type OrganizationIdentifier =
+  | { id: string; slug?: never } // Via id
+  | { id?: never; slug: string }; // Via slug
```

Comment:
❓ Is there a downside of supporting the case where _both_ organization id and slug and provided? Maybe to avoid dealing with the case where the slug and ID don't match?

Reply:
In my mind, the best thing would be to clearly communicate to the end-user that we need one or the other, ideally through the type system. 

I can't think of a reason why a caller would ever want to send both, apart from misunderstanding our docs and taking a belt-and-suspenders approach to specifying things. If we think that's likely to happen, I could see adding logic to check if they're both pointing to the same underlying org, and succeeding quietly. But that'd be my plan B!